__tests__/__unit__/shared/actions.unit.test.ts

```diff
@@ -35,41 +39,43 @@ async function createGlobalMocks() {
     Object.defineProperty(globals, "LOG", { value: jest.fn(), configurable: true });
     Object.defineProperty(globals.LOG, "debug", { value: jest.fn(), configurable: true });
     Object.defineProperty(globals.LOG, "error", { value: jest.fn(), configurable: true });
+
+    return globalMocks;
```

Comment:
Global mocks are different than the block ones. In the createGlobalMocks we are modifying things, effectively doing side effects. We should not be have return values in here. Also qpPlaceholder looks more like a constant than a global mock.
In block mocks on the other hand we are recreating everything from scratch and returning it. In there we should avoid doing side effects.
I would try to stick to this separation.
If you feel like it would be better to do this cleanup later, that is fine. There is an issue for it https://github.com/zowe/vscode-extension-for-zowe/issues/855

Reply:
Everything that is returned from the function is redeclared in the next it() block. This is why all the object definitions (other than globalMocks) have `configurable: true` in their assignment statements. The variable globalMocks is destroyed after every it() block and then redeclared in the next. What could be left over from this to cause side effects?